opendm/osfm.py

```diff
@@ -214,8 +214,8 @@ def setup(self, args, images_path, reconstruction, append_config = [], rerun=Fal
                 "bruteforce": "BRUTEFORCE"
             }
 
-            if not has_gps:
-                log.ODM_INFO("No GPS information, using BOW matching")
+            if not has_gps and not 'matcher_type_is_set' in args:
+                log.ODM_INFO("No GPS information, using BOW matching by default (you can override this by setting --matcher-type explicitely)")
```

Comment:
Minor typo.

explicitely should read explicitly.

Reply:
Uups! Thanks!